joomla/joomla-cms
# [4.0] Convert plugin system updatenotification to prepared statements

# Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

Check for update and try an older version of J4.


### Expected result
Nothing changed.
